ITMO_FS/utils/information_theory.py

```diff
@@ -7,54 +7,302 @@
 
 
 def conditional_entropy(x_j, y):
-    # H(Y|X)
-    buf = [[e[1] for e in g] for _, g in
+    """
```

Comment:
it seems better to create a separate test file for such measures

Reply:
i guess we can also do some unit tests in a separate file for these measures similar to how other algorithms have small doctests that we can verify by hand and a number of unit tests